odoo/fields.py

```diff
@@ -771,6 +771,11 @@ def resolve_depends(self, registry):
                     warnings.warn(f"Field {self} cannot be precomputed as it depends on non-precomputed field {field}")
                     self.precompute = False
 
+                if field_seq and not field_seq[-1]._description_searchable:
+                    # the field before this one is not searchable, so there is
+                    # no way to know which on records to recompute self
+                    warnings.warn(f"Field {field_seq[-1]} in dependency of {self} should be searchable")
```

Comment:
Shouldn't this be a `loggging.warn` instead?

And the message seems like it could be more helpful, maybe it should warn that the field is not searchable and explain what the consequences are? Possibly even explain the possible fixes and their tradeoffs?